Dockerfile

```diff
@@ -38,6 +38,7 @@ RUN cp bin/${APPNAME} /bin/${APPNAME}
 
 # Run image
 FROM registry.access.redhat.com/ubi8/ubi:latest
```

Comment:
If we are not using yum install, we can possibly change the image to minimal

Reply:
I think it requires more investigation.